deploy/skaffold/Dockerfile.deps

```diff
@@ -151,7 +151,8 @@ RUN gcloud auth configure-docker && gcloud components install --quiet \
     log-streaming
 
 FROM runtime_deps
-RUN apt-get update && apt-get install --no-install-recommends --no-install-suggests -y \
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get dist-upgrade -y && apt-get install --no-install-recommends --no-install-suggests -y \
```

Comment:
Can you add `curl` to the packages installed here? It seems this tool is not installed in Debian Bullseye but is in Ubuntu

This likely needs to be done in all of the Dockerfile.deps* files

Reply:
added

Reply:
When this got partially reverted in https://github.com/GoogleContainerTools/skaffold/pull/8460 it looks like this lost curl as we went from `gcr.io/gcp-runtimes/ubuntu_20_0_4` which included `curl` to `ubuntu:20.04` which doesn't.

Can `curl` be added back into the released Skaffold images please?

Reply:
@tomelliff thanks for flagging, this.  I've created this issue https://github.com/GoogleContainerTools/skaffold/issues/8668 tracking this and added it to Skaffold's next release -`v2.4.0` which is targeting release for 4/17